cartridges/app_custom_account_verification/cartridge/controllers/Login.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+/**
+ * @namespace Login
+ */
+
+var server = require('server');
+server.extend(module.superModule);
+
+var csrfProtection = require('*/cartridge/scripts/middleware/csrf');
+var consentTracking = require('*/cartridge/scripts/middleware/consentTracking');
+
+/**
+ * Login-Show : This endpoint is called to load the login page
+ * @name Base/Login-Show
+ * @function
+ * @memberof Login
+ * @param {middleware} - consentTracking.consent
+ * @param {middleware} - server.middleware.https
+ * @param {middleware} - csrfProtection.generateToken
+ * @param {querystringparameter} - rurl - Redirect URL
+ * @param {querystringparameter} - action - Action on submit of Login Form
+ * @param {category} - sensitive
+ * @param {renders} - isml
+ * @param {serverfunction} - get
+ */
+server.append(
+    'Show',
+    consentTracking.consent,
+    server.middleware.https,
+    csrfProtection.generateToken,
+    function (req, res, next) {
+        var viewData = res.getViewData()
+        viewData.accountCustomObject = req.querystring.accountCustomObject
```

Comment:
Why do we need this ?

Reply:
I fixed it as i use "status". I need this to extend view data to display message if user has already verified or the verification link has expired